### ISSUE TITLE
Inconsistent naming (win/pane)

### ISSUE THREAD

**Issue by EtienneDesticourt** [facebookresearch/visdom#162]
Functions take a win (window) argument but the documentation refers to windows as panes, and the window id returned is "pane_UID". 
It's inconsistent and confusing.

**Comment by lvdmaaten** [comment]
Good point. I would propose to adapt the phrasing in the documentation (rather than changing the `win` name, which would be a breaking code change).
